editor.planx.uk/docs/adding-a-new-component.md

```diff
@@ -135,23 +137,9 @@ case TYPES.SetValue:
 
 2. `src/@planx/components/shared/Preview/SummaryList`
 
-If/how should this component appear on the Review page?
+If/how should this component appear in a Review component:
 
 ```typescript
 [TYPES.SetValue]: undefined,
 ```
 
-3. `src/@planx/components/Send/bops/index`
-
-If/how should this component be formatted in Send data formats such as BOPS?
-
-```typescript
-function isTypeForBopsPayload(type?: TYPES) {
-  switch (type) {
-    // ...
-    case TYPES.SetValue:
-      return false;
-    // ...
-  }
-}
-```
```

Comment:
This was the only mention of the function `isTypeforBopsPayload` in the codebase, so I assume this is out of date!

Reply:
This has since been moved to planx-core which explains why you didn't find it, but is still a relevant step! https://github.com/theopensystemslab/planx-core/blob/main/src/export/bops/index.ts#L48